src/buildstream/buildelement.py

```diff
@@ -285,10 +297,19 @@ def configure_sandbox(self, sandbox):
             command_dir = build_root
         sandbox.set_work_directory(command_dir)
 
+    def stage(self, sandbox):
         # Setup environment
```

Comment:
I had to move this from `configure_sandbox()` to `stage()` because buildstream wouldn't let me stage dependency artifacts outside of `stage()`